### ISSUE TITLE
where to access the training  dataset? say, the Finger_ROI_Database.

### ISSUE THREAD

**Issue by dengyuhk** [wdongdongde/hardnet_triplet-for-Finger-Vein-Recognition#2]
Hello, I have no way to get the dataset Finger_ROI_Database in your implementation. And could you please release a link to this dataset?

**Comment by wdongdongde** [comment]
You can download some public datasets about finger vein.

**Comment by dengyuhk** [comment]
> You can download some public datasets about finger vein.

Yeah, I have downloaded some training data, it works. Thanks for your kind help, Sir.
